src/App/ComponentsDocumentation/components/Buttons/constants.js

```diff
@@ -83,6 +88,37 @@ const ButtonOptions = idAddOn => (
     }
 );
 
+const IconButtonOptions = idAddOn => (
```

Comment:
we're now only offering the size options for the icon buttons, not the Primary/secondary etc anymore,
is this on purpose?
bring it back I guess 😉

Reply:
What do you mean? 😅 It works for me?